src/Resources/app/storefront/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "finsearch",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "directories": {
+    "test": "test"
+  },
+  "scripts": {
+    "test": "jest"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/findologic/plugin-shopware-6.git"
```

Comment:
```suggestion
    "url": "https://github.com/findologic/plugin-shopware-6.git"
```

Reply:
Sure why not, it was autogenerated that way so I'd assume it would still be correct.